gen/flights/ai_flight_planner_db.py

```diff
@@ -174,6 +174,7 @@
     Su_25,
     F_15E,
     F_16C_50,
+    F_4E,
```

Comment:
These are a priority order, so putting it here means that the F-4 will be preferred for CAS over the F/A-18. I don't think that was your intent. Probably makes more sense below the Harrier?